user_docs/en/changes.t2t

```diff
@@ -30,6 +30,7 @@ What's New in NVDA
 == Bug Fixes ==
 - NVDA no longer fails to read focused controls in the Microsoft Account sign-in screen in Settings after entering an email address. (#7997)
 - NVDA no longer fails to read the page when going back to a previous page in Microsoft Edge. (#7997)
+- NVDA no longer fails to read items in list views within a 64-bit application such as Tortoise SVN. (#8175)

```

Comment:
Thanks for thinking of editing the changes file. We will actually update the changes file when we merge it to master, to prevent a ton of changes in it causing merge conflicts. Can you back this out for now?